compute/instances/troubleshooting/fix-lost-ip-connectivity-on-debian-buster.mdx

```diff
@@ -0,0 +1,32 @@
+---
+meta:
+  title: Fix lost IPv6 connectivity on old Debian Buster images when migrating to routed IP
+  description: This page helps recover or avoid unreachable IPv6 Instances after transitioning to routed IP on older Debian Buster images
+content:
+  h1: Fix lost IPv6 connectivity on old Debian Buster images when migrating to routed IP
+  paragraph: This page helps recover or avoid unreachable IPv6 Instances after transitioning to routed IP on older Debian Buster images
+tags: ipv6 routed ip debian buster
+dates:
+  validation: tbd
+  posted: tbd
+categories:
+  - compute
+---
+
+On old Debian Buster images, the installed custom version of cloud-init may interfere with the IPv6 connectivity when the instance has been transitioned to use routed IP. This may be avoided by installing a newer version of cloud-init prior to the migration to routed IP. The procedure is also valid to recover connectivity for an instance already using routed IP.
```

Comment:
```suggestion
On older Debian Buster images, the installed custom version of `cloud-init` may interfere with the IPv6 connectivity when the Instance has been transitioned to use routed IP. This may be avoided by installing a newer version of cloud-init prior to the migration to routed IP. The procedure is also valid to recover connectivity for an Instance already using routed IP.
```